### DISCUSSION TITLE
Multy tooltip

### DISCUSSION THREAD

**Discussion by Igogo-bus** [floating-ui/floating-ui#1428] [Q&A]
Hi. My problem is. I read few text strings from file (I don't know how many string I will red). I want create for each string new own tooltip. How I can do this? Thanks.

**Comment by FezVrasta** [comment]
Hi, the question is a bit too generic.

Where do you need to append these tooltips? Are these strings stored in an array once read? Are you using any specific JS framework?

**Comment by Igogo-bus** [comment]
Hi
Thank you for quick respond.
After transformation this strings will be convert to structure
< span id="line1" >Line 1< /span >
< span id="line2" >Line 2< /span >
Or something like above (will be different tags, e.g. < div > or < p> ). I need put tooltip on each line
< span id="line1" tooltip1 >Line 1< /span >
< span id="line2" tooltip2 >Line 2< /span >
I use jQuery framework

I need something like that
  $('[data-toggle="tooltip"]').tooltip({
   title:"",
   html:"true",
   viewport:"#putcodehere",
   trigger:"click hover focus",
   placement:"right"
  });
